src/citrine/resources/table.py

```diff
@@ -0,0 +1,73 @@
+from typing import Union
+
+import requests
+
+from citrine._rest.collection import Collection
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._serialization.properties import UUID
+from citrine._session import Session
+from citrine._utils.functions import rewrite_s3_links_locally, write_file_locally
+
+
+class Table(Resource['Table']):
+    """A 2-dimensional projection of data.
+
+    Tables are the basic unit used to flatten and manipulate data objects.
+    While data objects can represent complex materials data, the format
+    is NOT conducive to analysis and machine learning. Tables, however,
+    can be used to 'flatten' data objects into useful projections.
+    """
+
+    _response_key = 'table'
+
+    uid = properties.Optional(properties.UUID(), 'id')
+    version = properties.Optional(properties.Integer, 'version')
+    download_url = properties.Optional(properties.String, 'signed_download_url')
```

Comment:
These fields should be added to the class docstring as `Attributes` (see `Project` for an example)

Reply:
👍